.github/workflows/pr-labeler.yml

```
@@ -13,6 +16,16 @@ jobs:
       pull-requests: write
       issues: write
     steps:
-    - run: gh pr edit ${{ github.event.pull_request.number }} --add-label "auto-approve" -R ${{ github.repository }}
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: gh pr edit ${{ github.event.pull_request.number }} --add-label "auto-approve" -R ${{ github.repository }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  copy-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: kaizen3031593/pr-triage-manager@main
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
```

does this get treated differently?
```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

---

I suppose not. I don't think it matters, but I'll remove the quotes.